### ISSUE TITLE
Mispronunciation of Forbidden

### ISSUE THREAD

**Issue by Trix-Maiden** [espeak-ng/espeak-ng#441]
"Forbidden" has no stress on the second syllable ("ForBidden.") In this case, the pronunciation of the "for" part of the word is fine, but the lack of stress on the second syllable is incorrect. This affects "forbid" and other derivatives as well. "Forboding" has never had stress on the second syllable that I can remember, but it should.

**Comment by valdisvi** [comment]
I can't reproduce it. eSpeak NG says [f3b'Id@n](https://odo.lv/Espeak?eslang=-ven-gb&speed=160&output=-w&debug=-x&text=forbidden&formmethod=get) with stress on second syllable.
What version do you use, and do you use it directly, (e.g. from command line), or from some screen reader?

**Comment by rhdunn** [comment]
This is from NVDA with eSpeak 1.49.2, but I cannot reproduce the issue with forbidden on that version. It may happen when used in certain phrases/sentences. I have a local fix for forboding that I haven't committed yet.

**Comment by Trix-Maiden** [comment]
It happens wherever "Forbid", "Forbidden" and "forbidding" show up. I'm using the English Great-Britain voice if that helps.

**Comment by nmstoker** [comment]
I know it's a long time since this was last posted, but I also get **f'3bId@n** (ie not stressing the second syllable) and I'm using the command line (version 1.49.2 installed in Arch Linux installed from the [community package](https://www.archlinux.org/packages/community/x86_64/espeak-ng/)). It happens whether I use English RP or not:

```
espeak-ng -X -v gmw/en-GB-x-rp "forbidden"
espeak-ng -X "forbidden"
```
What I'm confused about is that the link @valdisvi provided back in 2018 is to the eSpeak NG testing page and yet that seems to use the latest commits. Thus, surely it's not surprising that the results there will differ, since aren't most users going to be on the released version of 1.49.2 from back in 2017?

Is there a new release planned as that would potentially clear up a number of queries like this? Then  downstream projects or users of the release version would get whatever changes have presumably been incorporated in the last ~2 years.

**Comment by hozosch** [comment]
Then you're glad to hear that it's fixed in espeak 1.49.3 dev. This issue can be closed I think. When is espeak 1.49.3 released btw?

**Comment by nmstoker** [comment]
Yes @hozosch that is good news and if there's a release soon that would be the icing on the cake :slightly_smiling_face:
